Extensions/tweaks.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE Tweaks **//
-//* VERSION 5.7.1 **/
+//* VERSION 5.7.2 **/
```

Comment:
needs version update

Reply:
to reduce the number of force-pushes i need to do, i'd prefer you look at #1702 before i do this